### PR TITLE
Automatic commit of package [pdc-contact-browser] release [2.0.0-1].

### PR DESCRIPTION
**Pull request by lubomir**
Created by command:

/usr/bin/tito tag --use-version 2.0.0

### CODE REVIEW

File: pdc-contact-browser.spec [hunk 1]
```diff
@@ -22,6 +22,41 @@ cp -R var/ %{buildroot}/
 /var/www/html/%{name}
 
 %changelog
+* Wed Jan 24 2018 Lubomír Sedlář <lsedlar@redhat.com> 2.0.0-1
+- Bump version (lsedlar@redhat.com)
```
Comment (hluk) [inline pdc-contact-browser.spec:26]: It would be better to clean up the changelog a bit. Not every commit is relevant to changes from last version.

**Comment by lubomir** [comment]
Removed the line from changelog. How would you feel about dropping tito completely? Then the release would just be bumping version in `package.json` and creating a tag (which could be just `make target`). The spec file in the repo is not really used anyway, so why bother updating it.

**Comment by hluk** [comment]
> How would you feel about dropping tito completely?

I'm OK with that.

**Comment by lubomir** [comment]
Ok, let's do that for next release.

**Review by hluk — APPROVED** [review]

**Pull request merged**